ethdb/README.md

```diff
@@ -0,0 +1,51 @@
+## ethdb
+
+Parity Ethereum interaction with the DB on-demand.
```

Comment:
Probably a dumb question: I'm not sure I understand what "on-demand" means in this context. Can you elaborate?

Reply:
i've explained it in the "Experience of New Feature" section of the notes here: https://hackmd.io/cXsUUCcDTQOm0ZF2KWQ6gg?view